Dockerfile

```diff
@@ -1,17 +1,17 @@
 # Build the manager binary
-FROM golang:1.16 as builder
+FROM registry.access.redhat.com/ubi8/go-toolset:latest as builder
 
-WORKDIR /workspace
+WORKDIR /opt/app-root/src
 COPY . .
 
 # Build
 RUN make build-operator
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
```

Comment:
I just re-reviewed, and see this -- I need to update my commit to remove this comment now we're using UBI instead.